docs/admin-manual/security.rst

```diff
@@ -212,14 +212,14 @@ If the condor daemon receiving the connection is the *condor_collector*, it firs
 checks to see if there are specific authorization lists for daemons advertising
 to the collector (i.e. joining the pool).  If the incoming command is
 advertising a submit node (i.e. a *condor_schedd* daemon), it will check
-``ALLOW_ADVERTISE_SCHEDD``.  If the incoming command is for an execute node (a
+:macro:`ALLOW_ADVERTISE_SCHEDD`.  If the incoming command is for an execute node (a
 *condor_startd* daemon), it will check ``ALLOW_ADVERTISE_STARTD``.  And if the
 incoming command is for a *condor_master* (which runs on all HTCondor nodes) it
-will check ``ALLOW_ADVERTISE_MASTER``.  If the list it checks is undefined, it will
-then check ``ALLOW_DAEMON`` instead.
+will check :macro:`ALLOW_ADVERTISE_MASTER`.  If the list it checks is undefined, it will
+then check :macro:`ALLOW_DAEMON` instead.
 
 If the condor daemon receiving the connection is not a *condor_collector*, the
-``ALLOW_DAEMON`` is the only list that is looked at.
+:macro:`ALLOW_DAEMON` is the only list that is looked at.
```

Comment:
I am unsure if the various ALLOW_* config options actually exist in the configuration macro docs

Reply:
The don't, but that's a problem for another day.